lib/spotlight/engine.rb

```diff
@@ -176,6 +176,17 @@ def self.blacklight_config
 
     Blacklight::Engine.config.inject_blacklight_helpers = false
 
+    config.i18n_locales = {
```

Comment:
I wonder if we could manage this as a larger project? It feels unwieldy to manage the translation native languages in the engine config. What do you think about a spin off ticket for a small gem that did this?

Reply:
Would it make sense for us to have a 2nd or 3rd use case before making a decision on that kind of abstraction?  I'm not sure we have enough information to know if this is something that would be served well be its own gem, and how one might want to interact with it/extend it.

FWIW, this was pulled directly from how Wikipedia links to its different language specific sites (as suggested by @ggeisler).

Reply:
Totally I get it, that is why I was just asking the question. It just seems arbitrary that we manage these native languages only for the locales that Blacklight provides. I def think this is the way to start this, but I see potential for others to use this also.

Reply:
Yeah, definitely agree, let's keep this in mind.  I agree it feels odd.